socialscan/cli.py

```diff
@@ -32,7 +32,8 @@
 async def main():
     start_time = time.time()
     colorama.init(autoreset=True)
-    sys.stdout.reconfigure(encoding='utf-8')
+    if sys.version_info >= (3, 7):
```

Comment:
Could you use a Python 3.6 compatible method of setting the stdout encoding to UTF-8? This will cause problems in certain cases when redirecting the output.